test/test_backends/test_s3.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, name):
         self.name = name
         self.keys = {}
 
-    def get_key(self, key):
+    def get_key(self, key, validate=True):
```

Comment:
The default does mimic the default, though nothing is done with it. Since this is test code, I could tack an `assert not validate` into this method